pootle/templates/browser/overview.html

```diff
@@ -75,10 +75,18 @@ <h4 class="clear">{{ item.title }}</h4>
 {% endblock %}
 
 {% block pre_content %}
-{% if announcement %}
+{% if announcement or upload_form or display_download %}
```

Comment:
@jleclanche @dwaynebailey I have tried to use overextends, but for some reason the overextend block is not being shown.

@julen Have you used overextends? I can't find any example on Pootle codebase.